A58-client-side-weighted-round-robin-lb-policy.md

```diff
@@ -121,12 +125,16 @@ In all languages, when a subchannel state is updated to `IDLE`, the LB policy im
 
 To enforce `blackout_period` and `weight_expiration_period`, the LB policy tracks two timestamps along with the weight. `last_updated` tracks the last time a report with non-zero `cpu_utilization` and `qps` was received. `non_empty_since` tracks when the first non-zero load report was received, and it is reset when a subchannel comes back to `READY` or the weight expires so that it can be updated with the next non-zero load report. The blackout and expiration are enforced when the weight value is looked up.
 
-The weight of a backend is calculated as ***weight = qps / cpuutilization***.
+The weight of a backend is calculated using `qps` (queries per second), `eps` (errors per second) and `cpu_utilization` reported by the backend. `eps` is only used when `error_utilization_penalty` is set to non-zero and applies a penalty based on the error rate calculated as `eps/qps`. The formula used is as follows:
+
+$$weight = \dfrac{qps}{cpu\_utilization + \dfrac{eps}{qps} * error\_utilization\_penalty}$$
```

Comment:
Looks like this isn't being formatted quite correctly.  The underscores are not being rendered, and the characters after the underscores are being rendered as subscripts.

Reply:
Updated. It looks in google3 and some formatters work without escaping \'s, but here we need to.